organization.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+@class Employee;
+@interface Organization:NSObject{
+    NSString *organizationName;
+    NSString *organizationId;
+    NSMutableArray *employeeList;
+}
+
+@property(retain)  NSString *organizationName;
+@property(retain)  NSString *organizationId;
+@property(retain)     NSMutableArray *employeeList;
```

Comment:
You can set this array to accept only the objects of class employee. Check how that's done.